js/advanced_search.js

```diff
@@ -17,6 +17,11 @@
         $('div.attachment-before').hide();
       }
 
+      $('label[for=edit-custom-897-2]').prepend('<span class="provider-icon icon_videoconferencing-img" title="' + Drupal.t('Online') + '"></span>');
+      $('label[for=edit-custom-897-3]').prepend('<span class="provider-icon icon_local_travel-img" title="' + Drupal.t('Online') + '"></span>');
```

Comment:
@monishdeb these titles need to be updated

Reply:
oops done